cmd/createtree/pem_test.go

```diff
@@ -0,0 +1,106 @@
+package main
+
+import (
+	"testing"
+
+	"github.com/golang/protobuf/ptypes"
+	"github.com/google/trillian/crypto/keys"
+	"github.com/google/trillian/crypto/keyspb"
+)
+
+func TestWithPEMKeyFile(t *testing.T) {
+	pemPath, pemPassword := "../../testdata/log-rpc-server.privkey.pem", "towel"
+
+	wantTree := *defaultTree
+	keyProto, err := ptypes.MarshalAny(&keyspb.PEMKeyFile{
+		Path:     pemPath,
+		Password: pemPassword,
+	})
+	if err != nil {
+		t.Fatalf("MarshalAny(PEMKeyFile): %v", err)
+	}
+	wantTree.PrivateKey = keyProto
+
+	runTest(t, []*testCase{
```

Comment:
It'll then lose access to the flags and won't be able to call `createTree()`. I can put them in their own file, but not their own package.